train.py

```diff
@@ -12,40 +20,36 @@
 from wavenet import WaveNet
 
 BATCH_SIZE = 1
-CHANNELS = 256
 DATA_DIRECTORY = './VCTK-Corpus'
-FILTER_WIDTH = 2
 LOGDIR = './logdir'
 NUM_STEPS = 1000
 LEARNING_RATE = 0.10
+NETWORK = './layout.json'
```

Comment:
Yes! I knew it wasn't great but couldn't think of anything better. `NETWORK_CONFIG`? `MODEL_CONFIG`? `WAVENET_CONFIG`? `WAVENET_PARAMS`?